### PR TITLE
enable same-name proposal on non hashed mappings

### PR DESCRIPTION
**Pull request by eristhea**
and delete a lot of "mark as deobf" mappings. i built this feature ages ago so we'd stop doing that and never turned it on
- removes "file not mapping a class" error. it's a good idea, but doesn't support name proposal

### CODE REVIEW

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping [hunk 1]
```diff
@@ -89,9 +84,8 @@ CLASS com/mojang/blaze3d/systems/RenderSystem com/mojang/blaze3d/systems/RenderS
 	METHOD enableBlend enableBlend ()V
 	METHOD enableColorLogicOp enableColorLogicOp ()V
 	METHOD enableCull enableCull ()V
-	METHOD enableDepthTest enableDepthTest ()V
 	METHOD enablePolygonOffset enablePolygonOffset ()V
-	METHOD enableScissor enableScissor (IIII)V
+	METHOD enableScissor (IIII)V
```
Comment (OroArmor) [inline mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:88]: The parameters on this swap the width and height. That seems wrong?

File: mappings/com/mojang/blaze3d/platform/GlStateManager.mapping [hunk 2]
```diff
@@ -293,78 +283,74 @@ CLASS com/mojang/blaze3d/platform/GlStateManager com/mojang/blaze3d/platform/GlS
...
 		ARG 0 id
-	METHOD setupGui3DDiffuseLighting setupGui3DDiffuseLighting (Lorg/joml/Vector3f;Lorg/joml/Vector3f;)V
+	METHOD setupGui3DDiffuseLighting (Lorg/joml/Vector3f;Lorg/joml/Vector3f;)V
 		ARG 0 light0
 		ARG 1 light1
-	METHOD setupGuiFlatDiffuseLighting setupGuiFlatDiffuseLighting (Lorg/joml/Vector3f;Lorg/joml/Vector3f;)V
+	METHOD setupGuiFlatDiffuseLighting (Lorg/joml/Vector3f;Lorg/joml/Vector3f;)V
 		ARG 0 light0
 		ARG 1 light1
-	METHOD setupLevelDiffuseLighting setupWorldDiffuseLighting (Lorg/joml/Vector3f;Lorg/joml/Vector3f;Lorg/joml/Matrix4f;)V
```
Comment (OroArmor) [inline mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:363]: This should stay

**Review by OroArmor — CHANGES_REQUESTED** [review]

File: mappings/com/mojang/blaze3d/platform/GlStateManager.mapping (hunk 2 above)
Comment (eristhea) [reply mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:363]: fixed

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:88]: reset all of them to be proposed based on the gl method

Comment (OroArmor) [reply mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:88]: Thanks. Looked weird from a quick glance

**Review by OroArmor — APPROVED** [review]

**Pull request merged**